app/controllers/application_controller.rb

```diff
@@ -4,10 +4,18 @@ class ApplicationController < ActionController::Base
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
   protect_from_forgery with: :exception
 
-  before_action :authenticate_user!
+  before_action :authenticate_user!, :banned?
```

Comment:
@igorkasyanchuk, if reporters will only be able to write issues and nothing else (it is important moment of this comment), as the owner of the project planned, I think it is no reason to keep blocked reporters signed in. Auto logout is pretty good way to block all abilities on the site. So, what do you think, this code must be changed anyway?